support/sier-codec/src/lib.rs

```diff
@@ -48,13 +50,79 @@ impl Parser {
     pub fn struct_def(&self, name: &str) -> Option<&Arc<StructDef>> {
         self.structs.values().find(|s| s.type_name() == name)
     }
-}
 
-impl Default for Parser {
-    fn default() -> Self {
-        Parser {
-            structs: HashMap::new(),
+    fn transform_serde_value<'a>(
+        &mut self,
+        value: &SerdeValue,
+    ) -> Result<(Type, Value<'a>), Error<'a>> {
+        match value {
+            SerdeValue::Null => Ok((Type::Unit, Value::Unit)),
```

Comment:
I don't think we want to parse null as (). I'd have to think through use cases to be sure, but it doesn't seem right.

Reply:
What should we parse it as? Maybe we need a Null type implementation?